rhizome/host/lib/boot_image.rb

```diff
@@ -0,0 +1,86 @@
+# frozen_string_literal: true
+
+require "digest"
+require "fileutils"
+require "uri"
+
+class BootImage
+  def initialize(name, version)
+    @name = name
+    @version = version || ""
+  end
+
+  def image_path
+    @image_path ||= if @version.empty?
+      "#{image_root}/#{@name}.raw"
+    else
+      "#{image_root}/#{@name}-#{@version}.raw"
+    end
+  end
```

Comment:
`nil` is a better indicator of a value's nonexistence.
```suggestion
  def initialize(name, version = nil)
    @name = name
    @version = version
  end

  def image_path
    @image_path ||= if @version
      "#{image_root}/#{@name}.raw"
    else
      "#{image_root}/#{@name}-#{@version}.raw"
    end
  end
```

Reply:
I did as you suggested, but didn't provide a default for `version`, at won't be optional after the transition to the new system.